dftimewolf/lib/processors/gcp_cloud_resource_tree_helper.py

```diff
@@ -266,54 +273,116 @@ def GenerateChildrenTree(self, level: int) -> List[Dict[str, 'Resource']]:
 
   def __str__(self) -> str:
     """Returns a string representation of the resource tree."""
-    output = '\n'
+    output = io.StringIO('')
+    # output = '\n'
     dashes = '-' * 200
-    # pylint: disable=line-too-long
-    output_format = '{:<20s}|{:<18s}|{:<20s}|{:<25s}|{:<18s}|{:<20s}|{:<25s}|{:<18s}|{:<100s}\n'
-
     # Draw table header
-    output = output + dashes + '\n'
-    output = output + output_format.format(
-        'Resource ID', 'Resource Type', 'Creation TimeStamp', 'Created By',
-        'Creator IP Addr', 'Deletion Timestamp', 'Deleted By',
-        'Deleter IP Addr', 'Tree')
-    output = output + dashes + '\n'
-
+    output.write(f'\n{dashes}\n')
+    output.write(f"""
```

Comment:
as i understand multiline strings include newlines/spaces in the string so where you have introduced this the output isn't the same as what has been replaced. checking that this is still WAI?

Reply:
Fixed